Gemfile

```diff
@@ -1,14 +1,10 @@
 source 'https://rubygems.org'
 #source 'https://rails-assets.org'
 
-ruby '~>2.1.2'
-gem 'rails', '~>3.2'
+ruby '~>2.6.5'
+gem 'rails', '~>4.2.8'
 
 # Database handling
-group :sqlite do
-  gem 'sqlite3'
```

Comment:
Aren't we using sqlite for production?

Reply:
No, SQLite is not used for production purposes, even if Spectral was a low-traffic website. See: https://stackoverflow.com/questions/913067/sqlite-as-a-production-database-for-a-low-traffic-site

Reply:
Okay cool :+1: